src/index.html

```diff
@@ -13,6 +13,40 @@
     />
```

Comment:
The stylesheet link should point to a CSS file, not a SCSS file. SCSS files need to be compiled to CSS before they can be used in HTML.